website/client/src/components/chat/autoComplete.vue

```diff
@@ -128,38 +130,29 @@ export default {
         backgroundColor: 'white',
       };
     },
-    searchResults () {
-      if (!this.searchActive) return [];
-      if (!this.atRegex.exec(this.text)) return [];
-      this.currentSearch = this.atRegex.exec(this.text)[0]; // eslint-disable-line vue/no-side-effects-in-computed-properties, max-len, prefer-destructuring
-      this.currentSearch = this.currentSearch.substring(1, this.currentSearch.length); // eslint-disable-line vue/no-side-effects-in-computed-properties, max-len
-
-      const lowerCaseSearch = this.currentSearch.toLowerCase();
-      return this.tmpSelections
-        .filter(option => { // eslint-disable-line arrow-body-style
-          return option.displayName.toLowerCase().indexOf(lowerCaseSearch) !== -1
-            || (option.username
-              && option.username.toLowerCase().indexOf(lowerCaseSearch) !== -1);
-        })
-        .slice(0, 4);
-    },
-
   },
   watch: {
-    text (newText) {
-      if (!newText[newText.length - 1] || newText[newText.length - 1] === ' ') {
+    text (newTextParam, prevTextParam) {
```

Comment:
It's to prevent conflicting naming schemes, since I noticed a few lines up (or down, I don't remember what), newText is a totally different meaning: it means text that the programmer generated, not the new text as part of the parameter of watch.